mappings/net/minecraft/data/server/recipe/StonecuttingRecipeJsonFactory.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3981 net/minecraft/data/server/recipe/SingleItemRecipeJsonFactory
+CLASS net/minecraft/class_3981 net/minecraft/data/server/recipe/StonecuttingRecipeJsonFactory
```

Comment:
Instead of renaming the class, rename the static methods instead. A different serializer can be passed to the constructor, like `CookingRecipeJsonFactory`.